src/components/general/MediumSettings.tsx

```diff
@@ -57,16 +57,16 @@ const StyledOutcomeCol = styled(Col)`
   }
 `;
 
-const MediumSettings = (props) => {
-  if (!process.browser) {
-    return null;
-  }
-  const site = useSite();
-  const instance = useInstance();
-
+const YearRangeSelector = (props) => {
+  const { minYear, maxYear, referenceYear } = props;
+  const inputReference = useRef(null);
   const [popoverOpen, setPopoverOpen] = useState(false);
   const toggle = () => setPopoverOpen(!popoverOpen);
 
+  // Focus on this when visible
+  useEffect(() => {
+    inputReference?.current?.focus();
+  }, []);
```

Comment:
Is this working as intended? When testing locally nothing was automatically focused

Reply:
Before you could not access the modal with keyboard at all. Now the focus goes to modal card itself so it's not visible as the card has no visual focus state.

Now you need to tab once more to focus on things inside the modal. Maybe we should add the ref to the maxYear button instead as that's the most likely thing to interact with. But then need to pass the ref a bit further...

Reply:
Ok this is not working anymore. I'm sure it did last night. Thanks for testing!! I'll debug